charts/kcp/values.yaml

```diff
@@ -3,34 +3,38 @@ etcd:
   enabled: true
   image: quay.io/coreos/etcd
   tag: v3.5.4
-  memoryLimit: 4Gi
+  cpuRequest: 500m
+  memoryLimit: 20Gi
+  memoryRequest: 2Gi
   volumeSize: 8Gi
   profiling:
     enabled: false
 kcp:
-  image: registry.ci.openshift.org/kcp/kcp
+  image: ghcr.io/kcp-dev/kcp
   tag: latest
-  v: "4"
+  v: "3"
+  logicalClusterAdminFlag: true
   memoryLimit: 20Gi
-  cpuRequest: 500m
-  memoryRequest: 4Gi
+  memoryRequest: 5Gi
+  cpuRequest: 100m
+  volumeClassName: gp2-csi
   etcd:
     serverAddress: https://etcd:2379
     clientCertificate:
       issuer: etcd-client-issuer
       commonName: root
   volumeSize: 1Gi
-  volumeClassName: ""
-  extraFlags: []  # Format is raw flag, e.g. --virtual-server-url=https://abc.xyz:443
+  extraFlags: []
   profiling:
-    enabled: false
+    enabled: true
     port: 6060
 kcpFrontProxy:
-  image: registry.ci.openshift.org/kcp/kcp-front-proxy
+  image: ghcr.io/kcp-dev/kcp
```

Comment:
I thought these images were dead and we were using the `registry.ci.openshift.org` ones. What gives?

Reply:
I was unaware, so I assumed these were the upstream images since they're mentioned in `https://github.com/kcp-dev/kcp/tree/main/manifest` and https://github.com/kcp-dev/kcp/pkgs/container/kcp appears current

The downstream images use a different (centos based) Dockerfile, so we need to decide which are preferred as the entrypoints are different

Reply:
This is another area for deprecation & consolidation

Reply:
There was some discussion on the community call with @ncdc, I think the next steps need to be something like:

- Confirm if we want to continue with [distroless](https://github.com/kcp-dev/kcp/blob/main/Dockerfile#L55) or switch to centos (which is currently used for the prow-built images)
- Add upstream Dockerfiles for each kcp component since the current Dockerfile doesn't include e.g syncer, we probably need several Dockerfiles similar to the [prow config](https://github.com/openshift/release/blob/master/ci-operator/config/kcp-dev/kcp/kcp-dev-kcp-main.yaml) inlined ones
- Update the prow config so the `registry.ci` images build from the upstream Dockerfiles
- Announce deprecation/removal of the ghcr.io images?

-

Reply:
>Announce deprecation/removal of the ghcr.io images?

One disadvantage to this which occurred to me is that the openshift images are likely less discoverable for community users, so for now I'll work on getting the images aligned, then we can discuss if we want to consolidate on a single registry or accept continuing with two locations for the same images.